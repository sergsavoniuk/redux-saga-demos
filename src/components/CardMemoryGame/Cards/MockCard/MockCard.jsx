import React from 'react';

import { Card } from '../Cards.components';

function MockCard() {
  return <Card active={false}>L</Card>;
}

export default MockCard;
