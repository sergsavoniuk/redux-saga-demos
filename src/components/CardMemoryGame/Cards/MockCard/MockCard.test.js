import React from 'react';
import { shallow } from 'enzyme';

import MockCard from './MockCard';

describe('<MockCard />', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<MockCard name="M" />);

    expect(wrapper).toMatchSnapshot();
  });
});
