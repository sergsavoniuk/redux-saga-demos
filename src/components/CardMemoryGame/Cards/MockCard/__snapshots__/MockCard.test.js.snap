// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MockCard /> renders correctly 1`] = `
<styled.div
  active={false}
>
  M
</styled.div>
`;
